api_tests/chronos/views/test_chronos_submission_detail.py

```diff
@@ -65,11 +65,11 @@ def test_update_failure(self, mock_update, app, url, submission, preprint_contri
         mock_update.return_value = submission
         payload = self.update_payload(submission)
         res = app.patch_json_api(url, payload, auth=preprint_contributor.auth, expect_errors=True)
-        assert res.status_code == 403
+        assert res.status_code == 404
```

Comment:
What's the reason for the update failure here? 404 seems weird if the user can see the preprint.